scripts/pre_commit_linter.py

```diff
@@ -112,7 +112,7 @@
 ]
 
 BAD_PATTERNS_FECONF = {
-    'FORCE_DEV_MODE = True': {
+    'FORCE_PROD_MODE = True': {
```

Comment:
One thought -- wouldn't it be better to positively verify the existence of the line "FORCE_PROD_MODE = False' instead? That way we keep this robust to issues like, e.g., `FORCE_PROD_MODE     = True`.

OK to do in a separate pass if you like, but please file an issue for this if that's the case.

Reply:
So if i understand this correctly is we are checking if the `FORCE_PROD_MODE = False` pattern is not in `feconf.py` to raise an error right?

Reply:
if is the case, will it be still named BAD_PATTERNS?

Reply:
> So if i understand this correctly is we are checking if the FORCE_PROD_MODE = False pattern is not in feconf.py to raise an error right?

Yes, that's right.

> if is the case, will it be still named BAD_PATTERNS?

No, probably the name will need to be changed, e.g. REQUIRED_STRINGS_FECONF. And perhaps the checker logic as well.

Reply:
DONE